test/common/crudtester.go

```diff
@@ -440,11 +440,7 @@ func (c *FederatedTypeCrudTester) checkHostNamespaceUnlabeled(client util.Resour
 			return false, nil
 		}
 		// Validate that the namespace is without the managed label
-		labels := hostNamespace.GetLabels()
-		if labels == nil || labels[util.ManagedByFederationLabelKey] != util.ManagedByFederationLabelValue {
-			return true, nil
-		}
-		return false, nil
+		return util.HasManagedLabel(hostNamespace), nil
```

Comment:
shouldn't this be `!util.HasManagedLabel(hostNamespace)` ?

Reply:
Done